web/locales/cs.yml

```diff
@@ -0,0 +1,6 @@
+cs:
+  RecurringJobs: Pravidelně opakované
```

Comment:
Hi @prokopsimek,

We prefer 'record_invalid'  for 'Record Invalid' if you don't mind.

Thanks